_data/sites.json

```
@@ -9187,6 +9187,16 @@
             "support.hotspotshield.com"
         ]
     },
+    {
+        "name": "Hotstar",
+        "url": "https://help.hotstar.com/in/en/support/tickets/new",
+        "difficulty": "medium",
```

If you need to open a ticket to support, then it sounds like `hard` fits better.
```suggestion
        "difficulty": "hard",
```